CONTRIBUTING.md

```diff
@@ -73,6 +73,17 @@ VisiData has two main branches:
 
 All pull requests should be submitted against `develop`. Submitters will need to e-sign a Copyright Assignment Agreement (CAA) before a pull request will be accepted.
 
+# Documentation and Help Text
+
+Follow these guidelines when writing documentation for VisiData, such as the help text for menu items:
+
+1. Use International English
+2. Avoid using pronouns. Use "you" or "they" where appropriate.
+3. Ensure that explanations are clear and precise. VisiData enables users to quickly query and change data in powerful ways, and our documentation should support that goal.
+4. Avoid assuming that users are familiar with command-line shells
+5. Avoid assuming that users are familiar with the Python programming language
```

Comment:
I do not actually agree with 4 and 5. I think there should be a part of the documentation that caters to users that are not familiar with command-line shells, and the Python programming language, but I think it is very fair to assume in the remainder a baseline of technical knowledge. For one thing, it will result in a ton of repetition throughout the documentation, and in helpstrings we cannot reall afford that. For another thing, I find that sometimes the use of simpler language, is less technically precise, and for me personally leads to more confusion. (See the example when a change was proposed to reword `null` to a less technically precise word that was subtly wrong.) Sacrificing comprehension for a theoreitcal accessibility makes the writing less accessible. I would rather we include a glossary, and educational materials, than throw it out completely. VisiData can be the gateway for someone gaining a comfort with the commandline, and Python. Users are smarter than given credit for here.

Reply:
This is a post that I would recommend us reading: http://jvns.ca/blog/confusing-explanations/#pattern-2-having-inconsistent-expectations-of-the-reader-s-knowledge

Inconsistent expectations are a huge problem, and so we should be clear at each set of documentation what our expectations are for the baseline reader's knowledge.

Reply:
I largely agree with @anjakefala on this.  I think there's room for us to provide beginner terminal and Python resources (whenever I look through @jsvine's tutorial I notice that the external link to basic Python expressions is not good for beginners, and I haven't been able to find a better one myself).  But it's outside the scope of the tool itself, as VisiData in general is optimized for terminal users and people with basic math/programming experience (if not Python specifically).

Reply:
I think we would definitely benefit from including more examples of the sorts of Python expressions you can do with VisiData! Some examples for `;` and `=` in particular would go really far.